spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubOperations.java

```diff
@@ -111,4 +125,15 @@
 	 * @return a received message, or {@code null} if none exists in the subscription
 	 */
 	PubsubMessage pullNext(String subscription);
+
+	/**
+	 * Deserializes the payload of the given message into the desired type.
+	 * @param pubsubMessage the message containing the payload data
+	 * @param objectType the class object of the desired type
+	 * @param <T> the desired type of the returned object
+	 * @return an instance of the object from the payload
+	 * @throws IOException if the payload could not be deserialized into the requested
+	 * type
+	 */
+	<T> T getPayloadFromMessage(PubsubMessage pubsubMessage, Class<T> objectType) throws IOException;
```

Comment:
I'm wondering if this should belong to `PubSubOperations` or just be a helper method in `PubSubTemplate`? WDYT?

Reply:
I'm fine with that. @meltsufin @artembilan does that sound ok?

Reply:
This doesn't look like a part of the `PubSubOperations` contract...

Reply:
done.